forge/forge/components/watchdog/watchdog.py

```diff
@@ -22,7 +22,7 @@ class WatchdogComponent(AfterParse[AnyProposal]):
 
     def __init__(
         self,
-        config: "BaseAgentConfiguration",
+        config: BaseAgentConfiguration,
```

Comment:
Does this work? I'm pretty sure Python will complain about that type being missing at runtime.

Reply:
Yes, thanks to `from __future__ import annotations`

Reply:
Ah yes, I hadn't expanded all of the file header and missed that import, my bad